lobby_server.yaml

```diff
@@ -8,5 +8,7 @@
     Please ensure you are using the latest stable release of Triplea located here http://www.triplea-game.org/
     Join the new testing group! For more info:
     https://forums.triplea-game.org/topic/268/volunteers-needed-early-release-testing-group
+    
+    https://forums.triplea-game.org/topic/69/revised-tournament-of-champions-11-toc-11  for intermediate or expert players only!
```

Comment:
Instead of the blank line, would it be more appropriate to have some kind of preface text like we did with the testing group?  The statement about player experience could be moved there, as well, instead of being at the end of the URL.  One possibility:

```
Revised Tournament of Champions 11 (for intermediate or expert players only!):
https://forums.triplea-game.org/topic/69/revised-tournament-of-champions-11-toc-11
```

Reply:
@ssoloff You can edit the file in @prastle's branch as well...
Go with whatever you find best